community/cypher/docs/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -37,6 +37,12 @@ include::exists.asciidoc[]
 
 Scalar functions return a single value.
 
+[IMPORTANT]
+The `LENGTH` and `SIZE` functions are quite similar, and so it is important to take note of the difference.
+Due to backwards compatibility `LENGTH` currently works on four types: strings, paths, collections and pattern expressions.
+However, for clarity it is recommended to only use `LENGTH` on strings and paths, and use the new `SIZE` function
+on collections and pattern expressions. `LENGTH` on those types may be deprecated in future.
```

Comment:
Checked the rendering in the docs, and it looked good. No issues related to line breaks. However, I am considering seeing if nicer rendering is possible with, for clarity.